include/swift/AST/DiagnosticsDriver.def

```diff
@@ -166,6 +166,9 @@ WARNING(warn_use_filelists_deprecated, none,
 ERROR(cannot_find_migration_script, none,
       "missing migration script from path '%0'", (StringRef))
 
+ERROR(error_darwin_static_stdlib_not_supported, none,
+      "-static-stdlib is no longer supported on darwin", ())
```

Comment:
Ah, I should have commented about this sooner: can you say "on Apple platforms" instead of "on darwin"?